examples/browser-js/build.gradle

```diff
@@ -1,76 +1,20 @@
 buildscript {
-    ext.kotlin_version = '1.3.11'
+    ext.kotlin_serialization_version = '0.20.0'
+    ext.protobuf_version = '3.11.1'
+    ext.pbandk_version = 'master-SNAPSHOT'
```

Comment:
You sure you want the snapshot version here?

Reply:
I was thinking that it'd be good for the examples to always be buildable against the current version on master. That way we can update them at the same time we update the main pbandk code and we can have CI build them to make sure they work.

When we do a release we'll want to temporarily set this to the release version for one commit. Then flip it back to the snapshot version.